packages/styles/scss/components/tabs/_tabs.scss

```
@@ -0,0 +1,880 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+//-----------------------------
+// Tabs
+//-----------------------------
+
+@use '../../config' as *;
+@use '../../theme' as *;
+// @use '../../themes' as *;
+
+@use '../../colors' as *;
+@use '../../type' as *;
+@use '../../breakpoint' as *;
+@use '../../spacing' as *;
+@use '../../motion' as *;
+
+@use '../../utilities/focus-outline' as *;
+@use '../../utilities/rotate' as *;
+@use '../../utilities/box-shadow' as *;
+@use '../../utilities/component-tokens' as *;
+@use '../../utilities/tabs' as *;
```

Unless other components are using these tokens, I think we should just move them to a `vars` file inside `tabs` 

```suggestion
@use 'vars' as *;
```